boosts.js

```diff
@@ -5286,7 +5286,7 @@ Molpy.DefineBoosts = function() {
 	{
 		if(times && Molpy.IsEnabled('Mario')) {
 			var l = Molpy.Boosts['Mario'].bought;
-			var runs = [10,20,100,500,1000][Molpy.options.approx];
+			var runs = 25;
```

Comment:
Because if the number was too big, it could crash computers, so 25 was safer. In general, I removed the option entirely. Besides, I used 25 in Photoelectricity and it didn't make sense to have different ones.